applications/StructuralMechanicsApplication/custom_elements/truss_element_linear_3D2N.cpp

```diff
@@ -268,6 +263,19 @@ double TrussElementLinear3D2N::ReturnTangentModulus1D(const ProcessInfo& rCurren
     KRATOS_CATCH("")
 }
 
+double TrussElementLinear3D2N::CalculateStress(const ProcessInfo &rCurrentProcessInfo)
+{
+    ConstitutiveLaw::Parameters Values(GetGeometry(),GetProperties(),rCurrentProcessInfo);
+
+    Vector temp_strain = ZeroVector(1);
+    Vector temp_stress = ZeroVector(1);
+    temp_strain[0] = CalculateLinearStrain();

```

Comment:
I would suggest a minor simplification:
```suggestion
    Vector temp_strain = ScalarVector(1, CalculateLinearStrain());
    Vector temp_stress = ZeroVector(1);

```